package.json

```diff
@@ -19,15 +19,15 @@
     "dotenv": "10.0.0",
     "lodash": "4.17.21",
     "uuid": "8.3.2",
-    "ws": "7.4.6"
+    "ws": "8.0.0"
   },
   "devDependencies": {
-    "@types/lodash": "4.14.170",
-    "@types/node": "14.14.41",
-    "@types/uuid": "8.3.0",
-    "@types/ws": "7.4.4",
+    "@types/lodash": "4.14.171",
+    "@types/node": "16.4.10",
```

Comment:
I think we should have 12.20.18 here, since the examples are supposed to work with node 12.x - but on the other hand the likelihood that we're using some node features accidentally from newer versions is very low.